client/components/SearchBar.tsx

```diff
@@ -0,0 +1,24 @@
+'use client'
+import { useState } from 'react';
+import TextField from '@mui/material/TextField';
+import { Container } from '@mui/material';
+
+export function SearchBar() {
```

Comment:
I would use consts for functions, it's the standard

Reply:
aight